### PR TITLE
couple small cleanups

### PR DESCRIPTION
**Pull request by mgeplf**
* use utils.load_json when possible
* check against collections.Mappings
* have pandas index match for edges tests

### CODE REVIEW

File: tests/test_edges.py [hunk 1]
```diff
@@ -120,15 +125,15 @@ def test_properties_1(self):
                 (0, 1, 88.1862, 1111.),
             ],
             columns=properties,
-            index=edge_ids
+            index=index(edge_ids)
```
Comment (mgeplf) [inline tests/test_edges.py]: when I ran the tests under py27/py35 and `pandas==0.24.2`, I was getting:

```
l = UInt64Index([1], dtype='uint64'), r = Int64Index([1], dtype='int64'), obj = 'DataFrame.index'

    def _check_types(l, r, obj='Index'):
        if exact:
>           assert_class_equal(l, r, exact=exact, obj=obj)
E           AssertionError: DataFrame.index are different
E
E           DataFrame.index classes are not equivalent
E           [left]:  UInt64Index([1], dtype='uint64')
E           [right]: Int64Index([1], dtype='int64')

.tox/py35/lib/python3.5/site-packages/pandas/util/testing.py:797: AssertionError

```

Comment (arsenius7) [reply tests/test_edges.py]: should we instead instruct `pandas.util.testing` to ignore `dtype` difference, if there is such an option?

Comment (mgeplf) [reply tests/test_edges.py]: yeah, I can use `check_index_type=False` I think, but I think it's actually good that it's unsigned, so worth checking

File: tests/test_edges.py [hunk 2]
```diff
@@ -19,6 +19,11 @@
 TEST_DATA_DIR = os.path.join(TEST_DIR, "data")
 
 
+def index(values):
```
Comment (arsenius7) [inline tests/test_edges.py]: `as_uint64`?

**Pull request merged**